hack/update-generated-crd.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -euo pipefail
+
+OUTDIR="${OUTDIR:-}"
+if [[ -z "$OUTDIR" ]]; then
```

Comment:
Could simplify:
```bash
if [[ -z "${OUTDIR-}" ]]; then
```

Reply:
I like it, but not enough to trigger a new round of CI tests